### PR TITLE
Create 2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md

### PR DESCRIPTION
**Pull request by solomon1923**
Adding a new page to the blog. Keeping it short as a test

### CODE REVIEW

File: _posts/2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: "$250 Bonuty - Be a guest on relevant podcasts and introduce the audience to bisq"
```
Comment (cbeams) [inline _posts/2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md:3]: Typo in the word that should be "Bounty"; Bisq should be capitalized.

File: _posts/2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+author: solomon1923
+excerpt: "Bisq v1.5.0 was released earlier today, bringing SegWit support across the trading protocol as well as a slew of updates to make trading better and more convenient. <br><br>"
```
Comment (cbeams) [inline _posts/2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md:5]: This excerpt was copied and pasted from the last blog post that went up in Nov 2020.

**Review by cbeams — CHANGES_REQUESTED** [review]
Fixes to the typos and other issues aside, I'd recommend against publishing this blog post. It just doesn't scan well that the first post in 3.5 years to the Bisq blog would be a bounty with a dollar figure attached to it for any random contributor to shill Bisq on a podcast. I'm sure this isn't how this is intended, but it's likely how it's going to look and feel to many who see it (myself included). It has the effect for me of cheapening the blog, which has never been used for such a purpose and comes off as sort of weak and desperate and suspicious.

As for the bounty itself, i.e. the GitHub issue, I'm less negative, but so far as I recall, we've never seen something like this work out before. There's just something fundamentally off with someone doing a podcast spot _because_ there was a bounty posted about it. It's always better, IMO, than they're (a) invited to do so because the podcast host sought them out or (b) they request to do the podcast because they're intrinsically motivated to do so, i.e. are just excited about Bisq, have recently contributed something themselves, etc. After the fact of the podcast going live, it's fine that they request some reasonable compensation for the effort, of course, but I'm skeptical that _motivating_ someone to take kind of action _in order_ to claim a bounty is basically getting things the wrong way around. I know that might leave you, as someone who is trying to promote growth of the Bisq network, seemingly having "your hands tied" and just waiting for someone to step up and do a podcast, but in my experience, these sorts of efforts have been ineffective and risk actually having negative impact because it looks (fairly or not) like we're trying to buy publicity for Bisq.

With that said, I commend the amount of thought that went into the design of the bounty, i.e. the requirements / criteria for satisfaction in the GH issue; I'm just not sure that marketing bounties like this are ever a good idea for Bisq.

**Comment by solomon1923** [comment]
Thanks @cbeams for the feedback.

The context of the PR request is I made a post on the community forum and Stacker News to promote the bounty and wanted to try and post a blog post to also promote it.

- https://bisq.community/t/bounty-250-usd-be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq/12609
- https://stacker.news/items/434641

Having not done a PR before, I was more focused on seeing if the blog could be uploaded from a PR than it's contents. I can see how this comes across as cheapening the blog. Apologies for this as it was not what I intended. 

With regards to whether bounties like this a good idea or not I share your skepticism but think it is a good idea to experiment with things to see what works. The background to the creation of the bounty is that I had 3 Bisq contributors reach out to me about interested in contributing to Bisq by taking about it on Podcasts. Rather than have three separate conversations in private I thought it better to create a bounty and open it up to the three contributors that expressed an interest as-well as anyone else that might be interested.

I will close this PR and try to another PR to update the blog once Bisq v1.9.15. I think all updates should be posted to the blog. Hopefully this will be a lot less controversial and will also enable me to go though my first PR process so I increase my confidence about being able to make updates to the blog. Thank you again for the constructive feedback.

**Comment by cbeams** [comment]
> I had 3 Bisq contributors reach out to me about interested in contributing to Bisq by taking about it on Podcasts.

Ah, that's great to hear. And all the more reason, I'd think, not to post about it on the blog, and simply to post the details and requirements of the bounty in a GH issue (as you did in bisq-network/growth#290) and then to point the interested parties to it.

In any case, sounds like a good plan to have the next post be about v1.9.15 when it's released, cheers!

**Pull request closed without merging**